dev/TabView/TabView.cpp

```diff
@@ -237,14 +237,20 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
         SelectedIndex(listView.SelectedIndex());
         SelectedItem(listView.SelectedItem());
 
-        m_scrollViewer.set([this, listView]() {
-            auto scrollViewer = SharedHelpers::FindInVisualTreeByName(listView, L"ScrollViewer").as<winrt::FxScrollViewer>();
-            if (scrollViewer)
+        auto scrollViewer = SharedHelpers::FindInVisualTreeByName(listView, L"ScrollViewer").as<winrt::FxScrollViewer>();
+        m_scrollViewer.set(scrollViewer);
+        if (scrollViewer)
+        {
+            if (SharedHelpers::IsIsLoadedAvailable() && scrollViewer.IsLoaded())
```

Comment:
We should avoid using ApiInformation for cases like this. Prefer QI as it's much faster (and will be happening anyway under the covers of the `.IsLoaded()` call).

```c++
if (winrt::IFrameworkElement7 scrollViewer7 = scrollViewer; scrollViewer7 && scrollViewer7.IsLoaded())
{
}
```